src/scss/constants.scss

```
@@ -1,11 +1,16 @@
-$primaryColor: var(--primary-color);
-$backgroundColor: var(--background-color);
+$primaryColor: var(--primary-color-deprecated);
+
+$primary-accent--over: var(--primary-accent--over);
+$primary-accent--under: var(--primary-accent--under);
+$primary-content--over: var(--primary-content--over);
+$primary-content-under: var(--primary-content--under);
+
+$primary--darken-20pp: var(--primary--darken-20pp);
+$primary--lighten-10pp: var(--primary--lighten-10pp);
```

Adjectives, not imperative verbs to describe nouns.

```suggestion
$primary--darker-20pp: var(--primary--darken-20pp);
$primary--lighter-10pp: var(--primary--lighten-10pp);
```

---

fixed in 5981d86e2f21047b93f3b7f639362c96520be00f